services/slam/fake/data_loader.go

```diff
@@ -4,35 +4,24 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"image"
 	"os"
 	"path/filepath"
 
 	"github.com/golang/geo/r3"
-	"github.com/pkg/errors"
 	"go.viam.com/utils"
 	"go.viam.com/utils/artifact"
 
-	"go.viam.com/rdk/pointcloud"
-	"go.viam.com/rdk/referenceframe"
-	"go.viam.com/rdk/rimage"
 	"go.viam.com/rdk/spatialmath"
-	rdkutils "go.viam.com/rdk/utils"
-	"go.viam.com/rdk/vision"
 )
 
-const chunkSizeBytes = 64 * 1024
```

Comment:
could we make this change in a separate pr?

Reply:
reverted to do in a separate PR, for this ticket https://viam.atlassian.net/browse/RSDK-2350

Reply:
ty!